modules/ROOT/pages/siebel/siebel-connector.adoc

```diff
@@ -156,13 +156,26 @@ Limit the number of objects to retrieve through DataSense to a few objects using
 
 == Log Requests and Responses [[log-requests-responses]]
 
-To log requests and responses when using the connector, configure the `log4j.xml` configuration file for the Mule app as follows:
+To log requests and responses when using the connector, add the following line to the `Loggers` element of the `log4j2.xml` configuration file for the Mule app:
```

Comment:
How about showing an example of the Logger block?

Reply:
Literally any log4j2.xml file of any connector or Mule application has that block. Here's an example file from the Service Bus connector for Mule 4:

```
<?xml version="1.0" encoding="UTF-8"?>
<Configuration status="WARN">
	<Appenders>
	    <Console name="Console" target="SYSTEM_OUT">
	    	<PatternLayout pattern="%d{HH:mm:ss.SSS} [%t] %-5level %logger{36} - %msg%n"/>
		</Console>
	</Appenders>
	<Loggers>
		<Logger name="org.mule.modules.microsoftservicebus" level="debug" additivity="false">
			<appender-ref ref="Console" level="debug"/>
		</Logger>
	</Loggers>
</Configuration>
```